smarts/core/agent_interface.py

```diff
@@ -202,6 +207,8 @@ class AgentInterface:
     The choice of action space, this action space also decides the controller that will be enabled.
     """
 
+    accelerometer: Union[Accelerometer, bool] = True
```

Comment:
If `Accelerometer` has no attributes, maybe just use a boolean flag, i.e., skip the `_resolve_config`

Reply:
In this case it is an option that could easily have more attributes in the future. I think it is better here as is.